src/main/java/org/opentripplanner/middleware/controllers/api/AbstractUserController.java

```diff
@@ -43,13 +47,20 @@ protected void buildEndpoint(ApiEndpoint baseEndpoint) {
         ApiEndpoint modifiedEndpoint = baseEndpoint
             // Get user from token.
             .get(path(ROOT_ROUTE + TOKEN_PATH)
-                .withDescription("Retrieves an " + persistence.clazz.getSimpleName() + " entity using an Auth0 access token passed in an Authorization header.")
-                .withResponseType(persistence.clazz),
+                    .withDescription("Retrieves an " + persistence.clazz.getSimpleName() + " entity using an Auth0 access token passed in an Authorization header.")
+                    .withResponseType(persistence.clazz),
                 this::getUserFromRequest, JsonUtils::toJson
             )
 
-            // Options response for CORS for the token path
-            .options(path(TOKEN_PATH), (req, res) -> "");
+            .get(path(ROOT_ROUTE + VERIFICATION_EMAIL_PATH)
+                    .withDescription("Triggers a job to resend the Auth0 verification email.")
+                    .withResponseType(Job.class),
+                this::resendVerificationEmail, JsonUtils::toJson
+            )
+
+            // Options response for CORS for the token and verification email paths
+            .options(path(TOKEN_PATH), (req, res) -> "")
+            .options(path(VERIFICATION_EMAIL_PATH), (req, res) -> "");
```

Comment:
@binh-dam-ibigroup, are these CORS methods needed anymore (now that we're using the proxy on API Gateway)?

Reply:
@br648 after doing a quick test on AWS, it looks like we don't need to add handlers for OPTIONS, so please remove them.
OPTIONS calls are (were already) forwarded by sparkjava to this code: https://github.com/ibi-group/otp-middleware/blob/dev/src/main/java/org/opentripplanner/middleware/OtpMiddlewareMain.java#L83

Reply:
Removed from AbstractUserController and ApiUserController